web-ui/src/main/resources/catalog/style/gn.less

```diff
@@ -889,6 +889,32 @@ gn-indexing-task-status {
   }
 }
 
+// fixed progress bar on top of the page
+.progress-bar-container {
+  height: 50px; // reserve height for the fixed bar
+  margin-top: -18px;
+  .progress-bar-container-fixed {
+    position: fixed;
+    z-index: 2;
+    padding-top: 15px;
+    background: #fff;
```

Comment:
isn't there a variable for this color?

Reply:
I guess I can use a variable, let me have a look